tests/cli/commands/test_cheat_sheet.py

```diff
@@ -18,7 +18,6 @@
 import contextlib
 import io
 import unittest
-from typing import List
```

Comment:
Let's try not to modify this file in this PR. This module tests the `cheat-sheet` command, not the `connection` command.